src/components/ItaliaTheme/View/Commons/SideMenu.jsx

```diff
@@ -163,7 +163,7 @@ const SideMenu = ({ data, content_uid }) => {
                   }}
                   aria-controls="side-menu-body"
                 >
-                  <h3>{intl.formatMessage(messages.index)}</h3>
+                  <p className="h3">{intl.formatMessage(messages.index)}</p>
```

Comment:
Anche qui come sopra per i correlati forse è bene che rimanga semanticamente un titolo, essendo una sezione con un ruolo dedicato (`nav`), che altrimenti dovrebbe avere credo un `aria-label`.

Reply:
Ho capito il discorso, però anche in questo caso sarebbe da mettere un h2, essendo una sezione? h2 con classe h3?

Reply:
fatto